CONTRIBUTING.md

```diff
@@ -0,0 +1,198 @@
+# Contributing to jobq
+
+Thank you for your interest in contributing to this project!
+
+We appreciate issue reports, pull requests for code and documentation,
+as well as any project-related communication through [GitHub Discussions](https://github.com/aai-institute/jobq/discussions).
+
+## Prerequisites
+
+We use the following development tools for all Python code:
+
+-   [uv](https://docs.astral.sh/uv/)
+-   [pre-commit](https://pre-commit.com/)
+-   [ruff](https://github.com/charliermarsh/ruff) (also part of the `pre-commit` hooks)
```

Comment:
This should be `astral-sh/ruff` IIUC.

Reply:
Ah yes, this was moved, I had copied from my browser history. Good catch!

Reply:
Fixed in 9d53c795df40bd9e51f959297b6093c77d7e4740